src/runtime/server/middleware/rate-limit.ts

```diff
@@ -8,17 +8,25 @@ import { useRuntimeConfig } from '#imports'
  * Only works on API routes, eg: /api/hello
  */
 export default defineEventHandler(async (event) => {
-  const isAPI = event.node.req.url?.includes('/api/')
-  const isEnabled = useRuntimeConfig().public.nuxtRateLimit.enabled
+  const { headers } = useRuntimeConfig().nuxtRateLimit
```

Comment:
If you only need data on the server, you should avoid using the `public` key. This will keep the client-side bundle slightly smaller